### PR TITLE
Add state on translation plateform in toDo table for letter translated not in Odoo

### PR DESCRIPTION
**Pull request by MickSandoz**

### CODE REVIEW

**Comment by ecino** [comment]
Ok for me

File: sbc_translation/models/import_letters_history.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    """
+
+    _inherit = 'import.letters.history'
+
+    @api.multi
+    def button_save(self):
+        """
+        save the import_line as a correspondence
+        """
+        logger.info("BUTTON SAVE ON CHILD !!!")
```
Comment (ecino) [inline sbc_translation/models/import_letters_history.py]: This seems a bit too apocalyptic :laughing:

Comment (MickSandoz) [inline sbc_translation/models/import_letters_history.py]: oops vieux commentaire, c'était quand je faisais des checks....

File: sbc_translation/models/import_letters_history.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        try:
+            super(ImportLettersHistory, self).button_save()
+        except Exception:
+            logger.info("EXCEPTION RAISE DURING IMPORT !!!")
```
Comment (ecino) [inline sbc_translation/models/import_letters_history.py]: Can you have something more precise and less dramatic ? Like "Exception during letter import" :wink:

File: sbc_translation/models/import_letters_history.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            # import done correctly so we can send to translate platform
```
Comment (ecino) [inline sbc_translation/models/import_letters_history.py]: This comment seems the opposite of what is actually happening... (?)

Comment (MickSandoz) [inline sbc_translation/models/import_letters_history.py]: Just i don't verify my comment...

File: sbc_translation/models/import_letters_history.py [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+            # letter to translate
+            for letters in self:
+                for corresp in letters.letters_ids:
+                    # logger.info("correspondence ? {}".format(c))
+                    if corresp.direction != "Beneficiary To Supporter":
```
Comment (ecino) [inline sbc_translation/models/import_letters_history.py]: Are you sure you can find B2S letters during an import ?

Comment (MickSandoz) [inline sbc_translation/models/import_letters_history.py]: Just, in this case i can remove this check

File: sbc_translation/models/import_letters_history.py [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+                        sponsorship = corresp.sponsorship_id
+                        original_lang = corresp.original_language_id
+                        if original_lang.translatable and original_lang not in\
```
Comment (ecino) [inline sbc_translation/models/import_letters_history.py]: Could you simply check `if corresp.state == 'GP Translation'` (I don't remember the exact state name) ?

**Pull request merged**